en/docs/administer/configuring-a-jdbc-user-store.md

```diff
@@ -41,25 +41,25 @@ Before you begin
 
 -   If you create the `           user-mgt.xml          ` file yourself,
     be sure to save it in the
-    `           <PRODUCT_HOME>/repository/conf/          ` directory.
+    `           <IS_HOME>/repository/conf/          ` directory.
 -   The `           class          ` attribute for JDBC is
     `           <UserStoreManager class="org.wso2.carbon.user.core.jdbc.JDBCUserStoreManager">          `
     .
 
 To configure a JDBC user store as the primary user store, you must
 change the `         JDBCUserStoreManager        ` section in the
-`         <PRODUCT_HOME>/repository/conf/user-mgt.xml        ` file.
+`         <IS_HOME>/repository/conf/user-mgt.xml        ` file.
 
 1.  Uncomment the following section:
 
-    ``` html/xml
+    ``` xml
     <UserStoreManager class="org.wso2.carbon.user.core.jdbc.JDBCUserStoreManager">
     ```
```

Comment:
I think this is templated. Check `https://github.com/pulasthi7/wso2-is-config/blob/master/userstores/jdbc_mysql_advance.toml`.